kvmagent/kvmagent/test/test_ft_vm_fencer.py

```
@@ -0,0 +1,212 @@
+
+for _p in (_kvmagent_root, _zstacklib_root):
+    if _p not in sys.path:
+        sys.path.insert(0, _p)
+
+# ---------------------------------------------------------------------------
+# 2. Mock heavy / Py2-only dependencies BEFORE any project import.
+#    Order matters: log → bash → everything else.
+# ---------------------------------------------------------------------------
+
+_mock_log = types.ModuleType('zstacklib.utils.log')
+_mock_log.get_logger = lambda name: mock.MagicMock()
```

Ruff (0.15.2)

[warning] Unused lambda argument: `name`

(ARG005)